docs/bundles/plain.md

```diff
@@ -29,16 +60,125 @@ manifests
 └── deployment.yaml
 ```
 
+For a bundle git repo, any directory that contains only static Kubernetes manifests checked into a git repository
```

Comment:
I got lost in this paragraph. What are you trying to say? The plain provisioner needs a `git` repository that supports the HTTP protocol?

Reply:
addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1 (hopefully this is a bit more clear)